openapi/openapi.json

```diff
@@ -896,6 +896,53 @@
                 },
                 "type": "object"
             },
+            "PspChannel": {
+                "properties": {
+                    "channel_code": {
+                        "type": "string"
+                    },
+                    "enabled": {
+                        "type": "boolean"
+                    },
+                    "payment_type_list": {
```

Comment:
[OTH](https://github.com/pagopa/pagopa-api/blob/1498e28a3f4d50ab2b462b12fcb33baf2558948c/general/PagInf_RPT_RT_6_2_0.xsd#L106) missing or not ? 
@corvettafabio